frontend/packages/knative-plugin/src/components/overview/RoutesOverviewList.tsx

```diff
@@ -4,28 +4,44 @@ import { K8sResourceKind } from '@console/internal/module/k8s';
 import { SidebarSectionHeading } from '@console/internal/components/utils';
 import { getKnativeRoutesLinks } from '../../utils/resource-overview-utils';
 import RoutesOverviewListItem from './RoutesOverviewListItem';
+import KSRouteSplitListItem from './KSRouteSplitListItem';
+import KSRoutesOverviewListItem from './KSRoutesOverviewListItem';
 
 export type RoutesOverviewListProps = {
   ksroutes: K8sResourceKind[];
   resource: K8sResourceKind;
 };
 
-const RoutesOverviewList: React.FC<RoutesOverviewListProps> = ({ ksroutes, resource }) => (
-  <>
-    <SidebarSectionHeading text="Routes" />
-    {_.isEmpty(ksroutes) ? (
-      <span className="text-muted">No Routes found for this resource.</span>
-    ) : (
+const RoutesOverviewList: React.FC<RoutesOverviewListProps> = ({ ksroutes, resource }) => {
+  if (_.isEmpty(ksroutes)) {
+    return (
+      <>
+        <SidebarSectionHeading text="Routes" />
+        <span className="text-muted">No Routes found for this resource.</span>;
+      </>
+    );
+  }
```

Comment:
will it make sense to have it as ternary as `<SidebarSectionHeading text="Routes" />` will always be there

Reply:
Updated.